docs/src/_autoapi_templates/python/module.rst

```diff
@@ -0,0 +1,119 @@
+{% if not obj.display %}
+:orphan:
+
+{% endif %}
+{% if obj.type is equalto("package") and obj.name is equalto("geovista") %}
+:fa:`file-lines` API Reference
+******************************
+
+{% endif %}
```

Comment:
This is the customisation.

Note that, `fontawesome` icons et al, are not rendered as aliases ref entries in a TOC, only within the source RST page.